sfm_pc/management/commands/import_google_doc.py

```diff
@@ -90,7 +104,7 @@ def get_conflicts(self, transpose=False):
         entity_map = self if not transpose else self.get_transposed()
 
         for key, values in entity_map.items():
-            if len(set(val for val, *_ in values)) > 1:
+            if len(set(val for val in values.keys())) > 1:
```

Comment:
this can be `if len(values) > 1` since values (as keys to a dictionary) are guaranteed to be unique

Reply:
Nice catch, thank you and donezo.